app/utils/goRoom.js

```diff
@@ -9,9 +9,9 @@ const navigate = ({ item, isMasterDetail, ...props }) => {
 	}
 
 	navigationMethod('RoomView', {
-		rid: item.rid,
+		rid: item.roomId || item.rid,
```

Comment:
Can't you call `goRoom` with rid?

Reply:
API response for `teams.create` does not bring rid, it brings roomId instead.

Reply:
`roomId` === `rid`. They are the same thing.